### PR TITLE
Show splash screen based on system theme

### PR DESCRIPTION
**Pull request by mr-manuel**

### CODE REVIEW

File: .github/patches/index.html.patch [hunk 1]
```diff
@@ -1,17 +1,27 @@
...
+     </style>
+   </head>
+   <body onload="init()">
 @@ -19,7 +19,7 @@
    <body onload="init()">
      <figure style="overflow:visible;" id="qtspinner">
        <center style="margin-top:1.5em; line-height:150%">
 -        <img src="qtlogo.svg" width="320" height="200" style="display:block"></img>
-+        <img src="victronenergy.svg" width="320" height="200" style="display:block"></img>
++        <img src="victronenergy-color-scheme.svg" width="320" height="200" style="display:block"></img>
```
Comment (jpetrell) [inline .github/patches/index.html.patch:18]: AFAICS victronenergy.svg is only used on the HTML splash screen, no need to have two images?

File: .github/patches/index.html.patch [hunk 2]
```diff
@@ -1,17 +1,27 @@
...
          <strong>Qt for WebAssembly: venus-gui-v2</strong>
          <div id="qtstatus"></div>
          <noscript>JavaScript is disabled. Please enable JavaScript to use this application.</noscript>
 @@ -46,6 +46,7 @@
                  status.innerHTML = 'Loading...';
- 
+
                  const instance = await qtLoad({
 +                    arguments: ['--mqtt', 'ws://' + document.location.host + ':9001/'],
```
Comment (jpetrell) [inline .github/patches/index.html.patch:26]: This does not seem related to the system color scheme changes.

File: .github/patches/index.html.patch (hunk 1 above)
Comment (jpetrell) [reply .github/patches/index.html.patch:18]: Old victronenergy.svg was mentioned in .github/patches/Makefile , I guess any new images and renames need to be adapted there.

File: .github/patches/index.html.patch (hunk 2 above)
Comment (mr-manuel) [reply .github/patches/index.html.patch:26]: No, probably because of latest main branch modifications.

Comment (jpetrell) [reply .github/patches/index.html.patch:26]: Ah yes true sorry getting confused with double plusses and minuses. :sweat_smile:

**Comment by mr-manuel** [comment]
Closed in favor of https://github.com/victronenergy/gui-v2/pull/1064.
